docs/can-fixture.md

```diff
@@ -34,7 +36,44 @@
   ```
   @codepen
 
-  When adding a fixture, it will remove any identical fixtures from the list of fixtures. The last fixture added will be the first matched.
+  The requestHandler also supports asynchronous results, by returning a promise or using `async`/`await`. This allows fixtures to depend on each other, introduce dynamic delays, and even depend on external resources.
```

Comment:
I would rewrite the first sentence to:

Return a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) (or use [async](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) & [await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await)) from `requestHandler` to asynchronously return results.